html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-syntax</title>
+  </head>
+  <body>
+    <img src="https://sportshub.cbsistatic.com/i/2023/01/21/db909e48-ea30-45f0-9639-79c5a22cf44f/pokemon-misty-psyduck-pikachu-reunion-funny.jpg" alt="" />
```

Comment:
Since this image actually has content, you would want to add a descriptive `alt` attribute here. Something like "psyduck". You only want to include an empty `alt` attribute, when the image has no relevance to the page. Something like a gray line that goes across the page for example